TanoaPeakSimulator/TanoaPeakSimulator/Tanoa.py

```diff
@@ -0,0 +1,92 @@
+import secrets
+import random
+
+
+def main():
+    location_length = 1000
+    bp_length = 100
+    bp_amount = 200
+
+    genome_file = open("dmel-all-chromosome-r6.38.fasta", "r")
+    sequences = genome_file.read()
+    sequences_arr = sequences.split('>')
+    sequence_object_array = []
+    for sequence in sequences_arr:
+        sequence_split = sequence.splitlines()
+        if len(sequence_split) != 0:
+            header = sequence_split[0]
+            body = ""
+            for x in range(1, len(sequence_split)):
+                body += sequence_split[x]
+            sequence_object_array.append(Chromosome(header, body))
+
+    # for chromosome in sequence_object_array:
+    #     for x in range(2000):
+    #         get_random_base_pair_string(location_length, bp_length, chromosome.sequence,
+    #                                     chromosome.bp_segment_arr)
+    # for chromosome in sequence_object_array:
+    #     chromosome.small_bp_segment_list = get_random_base_pair_string_list(chromosome.bp_segment_arr, bp_length,
+    #                                                                         bp_amount, chromosome.sequence)
+    chromosome = sequence_object_array[1]
+    for x in range(100):
+        get_random_base_pair_string(location_length, bp_length, chromosome.sequence, chromosome.bp_segment_arr)
+
+    chromosome.small_bp_segment_list = get_random_base_pair_string_list(chromosome.bp_segment_arr, bp_length,
+                                                                        bp_amount, chromosome.sequence)
+
+    generate_output_files(chromosome)
+
+
+def get_random_base_pair_string(sequence_length, bp_length, sequence, array_of_segments):
```

Comment:
I'm not sure what sequence length and bp length are.  Could you add comments to the function?

Reply:
mhm, I'll be adding comments to everything soon-ish

Reply:
Thanks! If the comments are converted to [docstring format](https://www.python.org/dev/peps/pep-0257/), then a user will be able to see them if they call `help(get_random_base_pair_string)` or if we were to add automatic document generation.